app/views/user/dashboard/show.html.erb

```diff
@@ -1,38 +1,66 @@
-<h1>Welcome <%= current_user.name %></h1>
-</div>
+<h1 class="white-text">Welcome <%= current_user.name %></h1>
+
 <div class="row">
-  <% @tanks.each do |tank| %>
-    <div class="col s12 m4">
-      <div class="card large">
-         <div class="card-image waves-effect z-depth-3 waves-block waves-light">
+  <div class="col s6 right">
+    <div class="grey darken-3 card large" style="border-radius: 20px">
+      <div class="card-content z-depth-3">
+        <%= render "/search/search_box" %>
+      </div>
+    </div>
+  </div>
+  <div class="col s6 card large left grey darken-3" style="border-radius: 20px; background-image: url('https://i1.wp.com/landoffish.com/wp-content/uploads/2017/03/Nitrate-chart-2.png?fit=610%2C382'); background-size: cover">
+  </div>
+</div>
+
+<div id="modal1" class="modal grey darken-3" style="border-radius: 20px">
+  <div class="modal-content">
+    <h4 class="orange-text">New Tank</h4>
+    <%= form_for @tanks.new_tank, url: "user/tank", method: :post do |f|%>
+    <%= f.label :name %>
+    <%= f.text_field :name %>
+    <div class="input-field col s12">
+      <select class="white-text grey darken-3">
+        <option value="" disabled selected>Choose your option</option>
```

Comment:
is this to allow them to start with none selected? or is place holder for planted?